features/LAY000_Layers-information.feature

```diff
@@ -0,0 +1,19 @@
+@implementer-agreement
+@LAY
+@version1
+@E00020
+
+Feature: LAY000 - Layers Information
+    The rule verifies the presence of IFC entities used to assign layers (also known as, CAD layer) to collection of elements. 
+    This is used mainly for grouping and visibility control, and in general to organise geometry into groups that may be shown or hidden.
+    https://ifc43-docs.standards.buildingsmart.org/IFC/RELEASE/IFC4x3/HTML/concepts/Product_Shape/Product_Geometry_Layer/content.html
+
+
+    Scenario: Check for activation
+
+    Given an IfcProduct
+    Given its attribute Representation
+    Given its attribute Representations
```

Comment:
There is one additional consideration, that's a bit in detail, but actually you can have layer assignment on the *representation* but also on the *items*. This is missing in the concept template, but is rendered in the documentation. 

![ifcpresentationlayerassignment-fig1](https://github.com/user-attachments/assets/4b803c9d-ad3d-43b0-bdac-cc31966a8b56)

I would actually add this, because semantically it makes more sense for me to assign the representation *item* than to assign the representation itself.

```
    Scenario: Layer assignment to representation

    Given an IfcProduct
    Given its attribute Representation
    Given its attribute Representations
    Given its attribute LayerAssignments

    Scenario: Layer assignment to representation items

    Given an IfcProduct
    Given its attribute Representation
    Given its attribute Representations
    Given its attribute Items
    Given its attribute LayerAssignments
```